README.md

```diff
@@ -21,18 +25,57 @@ This package is most easily installed by running
 pip install openmc-plotter
 ```
 
-
 ## Usage
 
-From a directory containing a working OpenMC model run:
+From a directory containing an OpenMC model run:
```

Comment:
What, you mean is doesn't work from a directory containing a broken model?! :laughing:

Reply:
Haha yeah. I felt like that didn't need to specified.